runtime/compiler/aarch64/codegen/CallSnippet.cpp

```diff
@@ -410,11 +410,18 @@ uint8_t *TR::ARM64VirtualUnresolvedSnippet::emitSnippetBody()
    TR_J9VMBase *fej9 = (TR_J9VMBase *)(comp->fe());
    void *thunk = fej9->getJ2IThunk(callNode->getSymbolReference()->getSymbol()->castToMethodSymbol()->getMethod(), comp);
 
+   // for alignment of intptr_t data
+   if (((uint64_t)cursor % TR::Compiler->om.sizeofReferenceAddress()) == 0)
```

Comment:
If your concern is about the size of `intptr_t` data, why not just use `sizeof(intptr_t)` here instead of the call to `sizeofReferenceAddress()` ?

Reply:
I agree it can be simply `sizeof(intptr_t)`.
`sizeofReferenceAddress()` came from the power version.